app/script/calling/CallingRepository.js

```diff
@@ -903,15 +903,26 @@ z.calling.CallingRepository = class CallingRepository {
   /**
    * Remove a participant from an call if he was removed from the group.
    *
-   * @param {string} conversationId - ID of conversation for which the user should be removed from the call
+   * @param {z.calling.entities.CallEntity} callEntity - Call entity
    * @param {string} userId - ID of user to be removed
    * @returns {undefined} No return value
    */
-  participantLeft(conversationId, userId) {
-    const additionalPayload = z.calling.CallMessageBuilder.createPayload(conversationId, this.selfUserId(), userId);
-    const callMessageEntity = z.calling.CallMessageBuilder.buildGroupLeave(false, this.sessionId, additionalPayload);
+  participantLeft(callEntity, userId) {
```

Comment:
Shouldn't it be `removeParticipant` here?

Reply:
Nice catch. You are absolutely right.